### PR TITLE
Update copyright statements to remove dates and ReadTheDocs footer

### PR DESCRIPTION
**Pull request by Kat-90**
As per the new recommendation from legal, the copyright statements no longer need a date in them. This PR removes all dates from the copyright statements and removes the yearly overhead of updating them.

Testing:
 - [X] Ran tests and they passed OK
 - [ ] Added new tests for the new feature(s)

CLA
 - [ ] If a new developer, signed up to CLA

### CODE REVIEW

**Comment by Kat-90** [comment]
New PR opened on clean branch to avoid conflicts.
Old PR with comments in: https://github.com/metoppv/improver/pull/1714

LICENCE file also updated - ran unit test and it passed.

ReadTheDocs copyright statement updated as shown here:

copyright

As far as I'm aware, I cannot remove the 'copyright' part at the beginning.

**Review by fionaRust — CHANGES_REQUESTED** [review]

File: improver/precipitation_type/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (Kat-90) [reply improver/precipitation_type/.idea/workspace.xml]: Thanks for spotting it. Removed and re-pushed.

**Comment by fionaRust** [comment]
@tjtg from our side we're happy for this to be merged, so feel free to merge it over our nighttime if you're happy from BoM side and know where it's going to interact with any of your work. Thanks

**Review by tjtg — APPROVED** [review]
Thanks for the updated PR - all looks good in this one. I'm going to go ahead and merge as suggested by @fionaRust.

**Pull request merged**